### PR TITLE
Replace `libssl3` with `libssl3t64`

### PR DESCRIPTION
**Pull request by pbusko**
## Summary

In Ubuntu Noble the `openssl` package now depends on `libssl3t64` instead of `libssl3`:

https://packages.ubuntu.com/noble/openssl

Also the location of the default apt sources is now in `/etc/apt/sources.list.d/ubuntu.sources` file, which uses `deb822` format

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: .github/workflows/create-release.yml [hunk 1]
```diff
@@ -352,7 +352,7 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.20.x
```
Comment (sophiewigmore) [inline .github/workflows/create-release.yml:355]: these changes get overwritten by whatever we have in https://github.com/paketo-buildpacks/github-config/tree/main/stack, but we should update this to be `stable` there anyway

Comment (sophiewigmore) [inline .github/workflows/create-release.yml:355]: can we use `'stable'` instead?

**Comment by sophiewigmore** [comment]
> Also the location of the default apt sources is now in /etc/apt/sources.list.d/ubuntu.sources file, which uses deb822 format

Hey @pbusko - how did you realize this?

**Comment by pbusko** [comment]
> > Also the location of the default apt sources is now in /etc/apt/sources.list.d/ubuntu.sources file, which uses deb822 format
> 
> Hey @pbusko - how did you realize this?

docker build logs are full of messages about duplicated apt sources, also the content of the `/etc/apt/sources.list` file is now this:

```bash
$ cat /etc/apt/sources.list
# Ubuntu sources have moved to the /etc/apt/sources.list.d/ubuntu.sources
# file, which uses the deb822 format. Use deb822-formatted .sources files
# to manage package sources in the /etc/apt/sources.list.d/ directory.
# See the sources.list(5) manual page for details.
```

**Review by sophiewigmore — APPROVED** [review]

**Pull request merged**